bolts/data/datasets/audio/ecoacoustics_ds.py

```diff
@@ -0,0 +1,121 @@
+"""Ecoacostics dataset provided A. Eldridge et al.
+    Alice Eldridge, Paola Moscoso, Patrice Guyot, & Mika Peck. (2018).
+    Data for "Sounding out Ecoacoustic Metrics: Avian species richness
+    is predicted by acoustic indices in temperate but not tropical
+    habitats" (Final) [Data set].
+    Zenodo. https://doi.org/10.5281/zenodo.1255218
+"""
+from __future__ import annotations
+import os
+from os import listdir
+from os.path import isfile, join
+from pathlib import Path
+from typing import ClassVar, Optional, Union
+import zipfile
+
+import numpy as np
+import pandas as pd
+import torch
+from typing_extensions import Literal
+
+from bolts.data.datasets.audio.base import PBAudioDataset
+from bolts.data.datasets.utils import AudioTform, FileInfo
+
+__all__ = ["EcoacousticsDS"]
+SoundscapeAttr = Literal["habitat", "site"]
+
+
+class EcoacousticsDS(PBAudioDataset):
```

Comment:
```suggestion
class Ecoacoustics(PBAudioDataset):
```
I think the `DS` suffix can be dropped -- the fact the class lives in `.datasets` makes it clear enough that it is a dataset.

Reply:
Good point, bit redundant, I just couldn't think of a catchy name like "NICO" or "MNIST" lol, will fix

Reply:
Fixed